sympy/diffgeom/diffgeom.py

```diff
@@ -499,17 +572,71 @@ def coord_tuple_transform_to(self, to_sys, coords):
 
     def jacobian(self, sys, coordinates=None):
         """
-        Return the jacobian matrix of a transformation.
+        Return the jacobian matrix of a transformation on given coordinates.
+        If coordinates are not given, coordinate symbols of *self* are used.
+
+        Parameters
+        ==========
+
+        sys : CoordSystem
+
+        coordinates : Any iterable, optional.
+
+        Returns
+        =======
+
+        sympy.ImmutableDenseMatrix
+
+        Examples
+        ========
+
+        >>> from sympy.diffgeom.rn import R2_r, R2_p
+        >>> R2_p.jacobian(R2_r)
+        Matrix([
+        [cos(theta), -rho*sin(theta)],
+        [sin(theta),  rho*cos(theta)]])
+        >>> R2_p.jacobian(R2_r, [1, 0])
+        Matrix([
+        [1, 0],
+        [0, 1]])
+
         """
         result = self.transform(sys).jacobian(self.symbols)
         if coordinates is not None:
             result = result.subs(list(zip(self.symbols, coordinates)))
         return result
-    jacobian_matrix = jacobian
+    jacobian_mat = jacobian_matrix = jacobian
```

Comment:
Is it necessary to have these shortcuts?

Reply:
probably not